src/control/ZoomControl.ts

```diff
@@ -249,6 +266,7 @@ class ZoomControl extends Component<ControlEvents> implements CameraControl, Opt
     targetEl.addEventListener(BROWSER.EVENTS.WHEEL, this._onWheel, { passive: false, capture: false });
     targetEl.addEventListener(BROWSER.EVENTS.TOUCH_MOVE, this._onTouchMove, { passive: false, capture: false });
     targetEl.addEventListener(BROWSER.EVENTS.TOUCH_END, this._onTouchEnd, { passive: false, capture: false });
+    targetEl.addEventListener(BROWSER.EVENTS.DOUBLE_CLICK, this._onDoubleClick);
```

Comment:
Unlike other events, does a double click need no additional options?

Reply:
`capture` needs to be false, so I guess it's fine without options.